hero-character-card-front/script.js

```diff
@@ -638,7 +666,16 @@ function parseAndDrawCardEffectBlock(block, index) {
   blockString = blockString.replaceAll('-', '–');
 
   // Extract all the words
-  let words = blockString.split(' ');
+  // add special processing for spaces after numbers
+  let words = blockString.split(' ').flatMap((word) => {
+    let newWord = word;
+    // double count the word afterwards
```

Comment:
I'm not the biggest fan of this double counting thing, but it was the simplest thing I could think of aside from rewriting the whole text rendering function. :)